src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -33,4 +36,24 @@ class RunWorkflowTests : SeleniumTest()
         Assertions.assertThat(tab.findElement(By.tagName("h2")).text).isEqualTo("Run workflow")
     }
 
+    @Test
+    fun `can select workflow progress tab and selecting a workflow option generates reports table`()
+    {
+        val link = driver.findElement(By.id("workflow-progress-link"))
+        Assertions.assertThat(link.text).isEqualTo("Workflow progress")
+        link.click()
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("workflow-progress-tab")))
+        val vSelectInput = driver.findElement(By.tagName("input"))
+        vSelectInput.sendKeys("workf")
+
+        val vSelect = driver.findElement(By.id("v-select"))
+        val dropdownMenu = vSelect.findElements(By.tagName("li"))
+        Assertions.assertThat(dropdownMenu[0].text).contains("workflow1\n" +
+                        "test.user@example.com | Tue Jun 15 2021, 14:50")
+        dropdownMenu[0].click()
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("workflow-table")))
+        val table = driver.findElement(By.id("workflow-table"))
+        Assertions.assertThat(table.text).contains("Reports")
```

Comment:
So it's getting as far as showing the table, so the status must be set, but there's nothing rendered in the table, i.e. don't seem to be any reports in the status? Is that because the status endpoint fetches status from orderly, and orderly doesn't know about the workflow - it's just had a row inserted into the OW db table? Might need to actually run the workflow through OrderlyServerAPI?

Reply:
Are there any examples of that having been done before that I can look at?

Reply:
No I don't think so, but I think it *should* "just work" if you create an [OrderlyServer](https://github.com/vimc/orderly-web/blob/master/src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt#L49) object and give it an [AppConfig](https://github.com/vimc/orderly-web/blob/master/src/databaseInterface/src/main/kotlin/org/vaccineimpact/orderlyweb/db/AppConfig.kt#L15) object as its parameter, I think it should read in the default config and be able to connect to orderly running in the dependencies.